package.json

```diff
@@ -44,7 +44,7 @@
   "peerDependencies": {
     "react": "^16.8.0",
     "react-dom": "^16.8",
-    "styled-components": "^2.x"
+    "styled-components": "^4.1.3"
```

Comment:
I think `^4.x` would be sufficient for the *peer dependency*. This looks more like the *dev dependency*.